### PR TITLE
SAS-60# add todos

### PR DESCRIPTION
**Pull request by AmitNachimovitz**

### CODE REVIEW

File: src/main/java/com/SAS/Controllers/systemController/systemController.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        System.out.println("Welcome to the setup of the soccer association system");
+    }
+
+    /**
+     * The function receives an external systems name and try to connect to this system
+     *
+     * @param name
+     */
+    public boolean addExternalSystem(String name) {
+        if (name == null) {
```
Comment (MrBahur) [inline src/main/java/com/SAS/Controllers/systemController/systemController.java:66]: maybe add ip address of system as parameter as well?

Comment (MrBahur) [reply src/main/java/com/SAS/Controllers/systemController/systemController.java:66]: or a way to get the ip by name (for example hashmap from name of system to ips)

Comment (MrBahur) [reply src/main/java/com/SAS/Controllers/systemController/systemController.java:66]: never mind, just saw the continue of the function and understood.

File: src/main/java/com/SAS/Controllers/systemController/systemController.java [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+     * The function receives userName and password from the user and creates admin user for the system
+     *
+     * @param userName
+     * @param password
+     * @param fullName
+     * @return
+     */
+    public boolean createSystemAdmin(String userName, String password, String fullName) {
+        if (userName == null || password == null || fullName == null) {
+            return false;
```
Comment (MrBahur) [inline src/main/java/com/SAS/Controllers/systemController/systemController.java:161]: maybe check the password/ admin/ fullname in different method, inorder to seperate between validaition and creation?

Comment (MrBahur) [reply src/main/java/com/SAS/Controllers/systemController/systemController.java:161]: for example, if we want to check that the password is "Strong enough" we need to add another if to the create, less modulary.

**Review by MrBahur — APPROVED** [review]

**Comment by MrBahur** [comment]
also, when you merge, please solve the conflict, its in the pom and i don't know what needed and what isn't so please look at it @AmitNachimovitz

File: src/main/java/com/SAS/Controllers/systemController/systemController.java (hunk 2 above)
Comment (AmitNachimovitz) [reply src/main/java/com/SAS/Controllers/systemController/systemController.java:161]: I added validate method and resolve conflicts in pom.xml

**Pull request merged**